include/fw_info.h

```diff
@@ -255,19 +259,19 @@ fw_find_firmware_info(u32_t firmware_address)
 }
 
 
-/* Check a fw_abi_info pointer. */
-static inline bool fw_abi_info_check(const struct fw_abi_info *abi_info)
+/* Check a fw_info_abi pointer. */
+static inline bool fw_info_abi_check(const struct fw_info_abi *abi_info)
 {
 	const u32_t abi_info_magic[] = {ABI_INFO_MAGIC};
-	return(memeq(abi_info->magic, abi_info_magic, CONFIG_FW_MAGIC_LEN));
+	return memeq(abi_info->magic, abi_info_magic, CONFIG_FW_INFO_MAGIC_LEN);
 }
 
 
 /* Expose ABIs to the firmware at this address. This is meant to be called
  * immediately before booting the aforementioned firmware since it will likely
  * corrupt the memory of the running firmware.
  */
-void fw_abi_provide(const struct fw_firmware_info *fw_info);
+void fw_info_abi_provide(const struct fw_info *fw_info);
 
 /* Get a single ABI.
```

Comment:
The API does not show up in the output. Probably because a lot of the comments start with /* instead of /**.

Reply:
Tried to fix this now.